src/transaction.rs

```diff
@@ -65,9 +86,16 @@ impl<'a> Transaction<'a> {
         Self {
             listeners: TxListeners::new(db, inner),
             done: false,
+            on_drop: OnTransactionDrop::Abort,
```

Comment:
Then, with `Default` being implemented on `OnTransactionDrop`:

```suggestion
            on_drop: OnTransactionDrop::default(),
```

Reply:
commit: [`77cd2ab361227c02b2949faaeb27ee9278b37e3b`](https://github.com/matrix-org/rust-indexed-db/pull/1/commits/77cd2ab361227c02b2949faaeb27ee9278b37e3b)